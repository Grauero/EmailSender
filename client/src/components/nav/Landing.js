import React from 'react';

const Landing = () => {
  const style = { textAlign: 'center' };

  return (
    <div style={style}>
      <h1>EmailSender</h1>
      Collect feedback from your users
    </div>
  );
};

export default Landing;
