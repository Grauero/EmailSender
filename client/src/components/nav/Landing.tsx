import React from 'react';

const Landing: React.FC<{}> = () => (
  <div className="landing">
    <h1>EmailSender</h1>
    <img src="mail.png" alt="app logo" width="300px" />
    <p>Collect feedback from your users</p>
  </div>
);

export default Landing;
