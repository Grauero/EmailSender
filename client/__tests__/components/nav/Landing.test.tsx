import 'jest';
import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Landing from '../../../src/components/nav/Landing';

const component = shallow(<Landing />);

test('matches snapshot', () => expect(toJSON(component)).toMatchSnapshot());
