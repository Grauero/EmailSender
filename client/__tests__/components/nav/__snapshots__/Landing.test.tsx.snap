// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="landing"
>
  <h1>
    EmailSender
  </h1>
  <img
    alt="app logo"
    src="mail.png"
    width="300px"
  />
  <p>
    Collect feedback from your users
  </p>
</div>
`;
